app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitTracksApiController.php

```diff
@@ -520,6 +520,7 @@ public function updateTrackBySummit($summit_id, $track_id)
                 'allowed_tags' => 'sometimes|string_array',
                 'allowed_access_levels' => 'sometimes|int_array',
                 'order' => 'sometimes|integer|min:1',
+                'proposed_schedule_transition_time' => 'sometimes|integer|min:1',
```

Comment:
this rule should be in add method too
also please move these rules to proper rules factory

Reply:
done